config/logger.js

```diff
@@ -1,17 +1,29 @@
 import { serverEnv } from "@config/schemas/serverSchema";
 import * as PinoLogger from "pino";
 
-const config = {
+const development = {
   transport: {
     target: "pino-pretty",
     options: {
+      level:'debug',
+      colorize: true,
+      translateTime:"SYS:dd-mm-yyyy HH:MM:ss",
+      ignore:"pid,hostname"   //Ignores process id and hostname
```

Comment:
Not following project coding standards, for example quotes and spacing

```suggestion
      level: "debug",
      colorize: true,
      translateTime: "SYS:dd-mm-yyyy HH:MM:ss",
      ignore: "pid,hostname", //Ignores process id and hostname
```